package-lock.json

```diff
@@ -107,7 +107,7 @@
         "react-native-web-linear-gradient": "^1.1.2",
         "react-native-web-lottie": "^1.4.4",
         "react-native-webview": "^11.17.2",
-        "react-native-x-maps": "1.0.9",
+        "react-native-x-maps": "^1.0.9",
```

Comment:
This must be some change from some other PR, can you confirm you get this diff on main as well and if so lets just leave it her @s77rt

Reply:
Not in main

Reply:
After running npm install on main the package-lock will update the version as `^1.0.9`. We can leave it.